src/java/com/phenix/pct/AnalyticsBuildListener.java

```diff
@@ -0,0 +1,90 @@
+package com.phenix.pct;
+
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.ResourceBundle;
+import java.util.Map.Entry;
+
+import org.apache.tools.ant.BuildEvent;
+import org.apache.tools.ant.BuildListener;
+import org.apache.tools.ant.Task;
+
+public class AnalyticsBuildListener implements BuildListener {
+    private static final Map<String, Integer> taskCount = new HashMap<>();
+
+    @Override
+    public void taskFinished(BuildEvent event) {
+        Task task = event.getTask();
+        if (task == null) {
+            return;
+        }
+        String type = task.getTaskType();
+        if (type == null) {
+            return;
+        }
+        if (type.toLowerCase().startsWith("pct") || "RestGen".equalsIgnoreCase(type)
+                || "DlcHome".equalsIgnoreCase(type) || "Sports2000".equalsIgnoreCase(type)
+                || "DlcVersion".equalsIgnoreCase(type) || "ABLUnit".equalsIgnoreCase(type)
+                || "ClassDocumentation".equalsIgnoreCase(type)) {
+            Integer i = taskCount.get(type.toLowerCase());
+            if (i == null)
+                taskCount.put(type.toLowerCase(), 1);
+            else
+                taskCount.put(type.toLowerCase(), i + 1);
+        }
+    }
+
+    @Override
+    public void buildFinished(BuildEvent event) {
+        StringBuilder sb = new StringBuilder("pct version=\"");
+        sb.append(ResourceBundle.getBundle(Version.BUNDLE_NAME).getString("PCTVersion")).append('"');
+        for (Entry<String, Integer> entry : taskCount.entrySet()) {
+            sb.append(',').append(entry.getKey() + "=" + entry.getValue());
+        }
+        System.out.println("str : " + sb.toString());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS106)